pkg/networkservice/mechanisms/vxlan/client.go

```diff
@@ -75,8 +67,8 @@ func (v *vxlanClient) appendInterfaceConfig(ctx context.Context, conn *networkse
 			Enabled: true,
 			Link: &vppinterfaces.Interface_Vxlan{
 				Vxlan: &vppinterfaces.VxlanLink{
-					SrcAddress: dstIP,
-					DstAddress: srcIP,
+					SrcAddress: mechanism.SrcIP().String(),
```

Comment:
Is there any chance that SrcIP or DstIP won't exist?

Reply:
Worst case scenario srcIP and dstIP are "", which will lead to an error when we do our commit to vppagent.